conf/test.config

```diff
@@ -44,4 +44,6 @@ params {
   input_sheet_dda = 'https://raw.githubusercontent.com/nf-core/test-datasets/diaproteomics/dda_sheet.tsv'
   input = 'https://raw.githubusercontent.com/nf-core/test-datasets/diaproteomics/sample_sheet.tsv'
   unimod = 'https://raw.githubusercontent.com/nf-core/test-datasets/diaproteomics/unimod.xml'
+  // Ignore `--input` as otherwise the parameter validation will throw an error
+  schema_ignore_params = 'genomes,input_paths,input'
```

Comment:
As with other review, I don't think that you're using  `input_paths` (or `genomes`?)